### ISSUE TITLE
Filter or find drawables

### ISSUE THREAD

**Issue by shiraji** [androhi/AndroidDrawableViewer#11]
Is there a way to filter/find drawables?
I have no luck to find an easy way to implement this feature.

**Comment by androhi** [comment]
I have just released version 1.3 now. It include feature to filter drawable folder.
If you differ with your image, please let me know.

**Comment by shiraji** [comment]
No. What I want is the cmd+f function. One of my android projects has more than 100 of drawables in each folder. Filtering drawable folder function won't help the project. 

On the "Project" tool window
1. type cmd+f
2. you will see "Search for:" popup
3. type "keyword"
4. highlight the file that contains "keyword"

**Comment by androhi** [comment]
Sorry, I understand it.
What a cool function it is! I'll do my best.
